cypress/fixtures/listOfTwoSwaps.json

```diff
@@ -1,62 +1,102 @@
 {
-  "_embedded": {
-    "swaps": [
-      {
-        "_links": {
-          "self": {
-            "href": "/swaps/rfc003/ab02b94f-b66c-48a3-8ed3-1e300391170e"
-          }
-        },
+  "class": [],
+  "properties": null,
+  "entities": [
+    {
+      "class": [],
+      "properties": {
         "parameters": {
           "alpha_asset": {
-            "name": "Bitcoin",
+            "name": "bitcoin",
             "quantity": "100000000"
           },
           "alpha_ledger": {
-            "name": "Bitcoin",
+            "name": "bitcoin",
             "network": "regtest"
           },
           "beta_asset": {
-            "name": "Ether",
-            "quantity": "10000000000000000000"
+            "name": "ether",
+            "quantity": "1000000000000000000"
```

Comment:
🙃 not that it should make a difference, but I wonder why the quantity changed ;-)

Reply:
Haha, well, instead of manually adding the swaps I just copied the json the comit node returns after sending two swap requests <:^)